java/example.conf

```diff
@@ -0,0 +1,32 @@
+# This is an example ray config file.
+# To use this file, copy it to your classpath and rename it to 'ray.conf'.
+
+# For all available config items, see 'java/runtime/src/main/resources/ray.default.conf'.
+# For config file format, see 'https://github.com/lightbend/config/blob/master/HOCON.md'.
+
+ray {
+
+  // Path of your Ray home dir,
+  // can be either an absolute path, or a relative path to current work dir.
+  home = "/path/to/your/ray/home"
+
+  // Worker's mode, available options are:
+  // `WORKER`: indicates that this worker is a normal worker.
+  // `DRIVER`: indicates that this worker is a driver.
+  worker.mode = DRIVER
```

Comment:
Under what circumstances should this be set to `WORKER` or `DRIVER`?

Reply:
`WORKER` should only be set in worker process (the process forked from worker_pool.cc). Actually I think this shouldn't be exposed to user.

Reply:
I'm a bit confused about this. To run an application, does the user need to make one conf file for the cluster, one for the application, or one for every worker?

Reply:
Yep. I think we should add an API to specify the worker mode:
For a normal worker, we can invoke this:
```java
Ray.init(WorkerMode.WORKER);
```
And call `ray.init()` without argument for a driver.

Reply:
It's unclear to me what `Ray.init(WorkerMode.WORKER);` means. Under what circumstances would someone do that? Just for testing/debugging?

Reply:
User should call `Ray.init()` instead of `Ray.init(WorkerMode.WORKER)`.
Because `Ray.init()` will specify the worker mode as DRIVER.
`Ray.init(WorkerMode.WORKER)` just for the [`DefaultWorker.java`](https://github.com/ant-tech-alliance/ray/blob/simplify_config/java/runtime/src/main/java/org/ray/runtime/runner/worker/DefaultWorker.java#L18).

And then, we can remove the worker mode item from config file.

Reply:
> I'm a bit confused about this. To run an application, does the user need to make one conf file for the cluster, one for the application, or one for every worker?

there should be only one file for the application. And worker processes should use the same file, except overriding `ray.worker.mode=WORKER` (see `DefaultWorker.java`). 

Thus, users always call `Ray.init()`, and don't need to worry about which mode they should set.